cmd/edenUtils.go

```diff
@@ -178,15 +178,13 @@ func newTemplateCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			tmpl, err := os.ReadFile(args[0])
-			if err != nil {
-				log.Fatal(err)
-			}
-			out, err := utils.RenderTemplate(cfg.ConfigFile, string(tmpl))
-			if err != nil {
+			var out bytes.Buffer
+
+			if err := openevec.ParseTemplateFile(args[0], *cfg, &out); err != nil {
```

Comment:
Curious, what exactly wrong with old function? I can see that you re-implement it for some reason.

Reply:
it was using map with functions to fetch from Viper. We want to use viper only in CLI, new function uses configuration file directly to parse and write template

Reply:
Fair, this can stay as it was. Will revert this commit